CHANGELOG.txt

```diff
@@ -17,6 +17,11 @@ master:
    * Added __eq__ to QuantityError so empty instances equal None (see #2185).
    * Reworked the event scoped resource identifiers for the event classes
      hopefully fixing all edge-cases (see #2091).
+   * Fixed issue where resource_ids could refer to incorrect object if the
+     original object went out of scope and a new object shared the python
+     id of the old object (see #2278).
```

Comment:
If I understand correctly, this problem was only introduced on master and it's not in any release version, so I think we don't need a bugfix comment?

Reply:
Correct, will remove it.